Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -1471,6 +1480,7 @@ private void loadskin()
                 }
             }
             Camera.main.GetComponent<SpectatorMovement>().disable = true;
+            Camera.main.GetComponent<Assets.Scripts.UI.Camera.MouseLook>().disable = true;
```

Comment:
Import the namespace, and use MouseLook instead of Assets.Scripts.UI.Camera.MouseLook